src/components/index.ts

```diff
@@ -36,4 +35,5 @@ export * from "./Table";
 export { TabContent, Tabs, TabsWithContent } from "./Tabs";
 export type { RouteTab, RouteTabWithContent, Tab, TabWithContent } from "./Tabs";
 export * from "./Tag";
+export * from "./Toast/Toast";
```

Comment:
Suggestion: Maybe do:

```
export { showToast } from "./Toast/useToast";
```

As I think that's the primary user-facing API for setting the toast.